test/compilable/test8509.d

```diff
@@ -1,3 +1,11 @@
+/*
+TEST_OUTPUT:
+---
+compilable/test8509.d(15): Deprecation: `opCat` is deprecated.  Use `opBinary(string op)(...) if (op == "~")` instead.
+compilable/test8509.d(16): Deprecation: `opCat` is deprecated.  Use `opBinary(string op)(...) if (op == "~")` instead.
```

Comment:
Likewise, replace with new operator syntax, the issue wasn't about operators: https://issues.dlang.org/show_bug.cgi?id=8509